### PR TITLE
Add @carl-christofferson bio

### PR DESCRIPTION
**Pull request by carl-christofferson**
This Pull Request adds a bio for @carl-christofferson as requested.

### CODE REVIEW

File: Bios/carl-bio.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+##Carl Bio##
+**Name:** Carl
+
+**Occupation:** Software Engineer
+
+**Location:** San Diego
```
Comment (carl-christofferson) [inline Bios/carl-bio.md]: Is this the one in CA? Maybe be more specific

**Comment by jungao1999** [comment]
This is a testing general comment

File: Bios/carl-bio.md (hunk 1 above)
Comment (jungao1999) [inline Bios/carl-bio.md]: This is a testing in-line comment

**Comment by carl-christofferson** [comment]
:+1: This looks good now

**Pull request merged**